src/app/shared/kpm-details-form/kpm-details-form.component.html

```diff
@@ -89,14 +89,17 @@
                 <div class="input-group">
                     <input type="number" class="form-control" [(ngModel)]="keyPerformanceMetric.baselineValue"
                         name="{{'baselineValue_'+keyPerformanceMetric.value}}" (input)="calculateCost(false)"
-                        [disabled]="disableForm">
+                        [disabled]="disableForm" min="0" (blur)="validatePositiveValue('baselineValue')">
                     <span class="input-group-text">
                         <span [innerHTML]="keyPerformanceMetric.totalUnit | unitsDisplay"></span>/yr
                         <!--TODO: Issue 231-->
                         <!-- <span
                             [innerHTML]="keyPerformanceMetric.timePeriod | unitsDisplay"></span> -->
                     </span>
                 </div>
+                <div class="alert alert-warning form-error" *ngIf="keyPerformanceMetric.baselineValue < 0">
+                    Value can not be negative.
```

Comment:
We need to decide if these are warnings or errors. If it's a warning this should be "Value should not be negative" and we don't want to overwrite the value entered.

If it's an error, we should have "alert-danger" and I still wouldn't overwrite the number but have the user fix it before leaving the page with a route guard. I don't think we should overwrite the number to 0 if it is negative. From experience, this ends up more of an issue than a help for users